ci/cron/src/Main.hs

```diff
@@ -274,24 +287,18 @@ check_releases bash_lib = do
 
               "shopt -s extglob", -- enable !() pattern: things that _don't_ match
 
-              -- TODO: get all releases (GH paginates by 30)
-              "RELEASES=$(curl https://api.github.com/repos/digital-asset/daml/releases -s)",
-              "for i in $(seq 1 $(echo \"$RELEASES\" | jq length)); do",
-                  "VERSION=$(echo \"$RELEASES\" | jq -r \".[$i-1].tag_name\")",
-                  "mkdir \"$VERSION\"",
-                  "cd \"$VERSION\"",
                   "PIDS=\"\"",
-                  "for ass in $(seq 1 $(echo \"$RELEASES\" | jq \".[$i-1].assets | length\")); do",
+                  "for ass in " <> Data.List.intercalate " " (assets release) <> "; do",
```

Comment:
seems like this whole block maybe should be deindented now that it’s no longer in the loop.

Reply:
Yes, but that made the diff really horrible to read.

Reply:
Though [ignoring whitespace changes](https://github.com/digital-asset/daml/pull/7586/files?w=1) recovers some of the readability.

Reply:
Fair although github has an option to hide whitespace changes. That said, I don’t care at all about doing this in this PR or doing it at all given that I hope most of this bash script disappears in the future anyway.